src/domains/fdr-app/00_alert_fdr_fase1.tf

```diff
@@ -0,0 +1,69 @@
+// PROD AzureDiagnostics url_s operationId_s
+// UAT ApiManagementGatewayLogs Url OperationId
+resource "azurerm_monitor_scheduled_query_rules_alert" "alert-fdr-nodo-auth-error" {
+  for_each            = { for c in local.api_fdr_nodo_auth_alerts : c.operationId_s => c }
+  name                = "fdr-nodo-auth-${each.value.primitiva}-app-exception"
+  resource_group_name = data.azurerm_resource_group.fdr_rg.name
+  location            = var.location
+
+  action {
+    action_group           = var.env_short == "p" ? [data.azurerm_monitor_action_group.email.id, data.azurerm_monitor_action_group.slack.id, data.azurerm_monitor_action_group.opsgenie[0].id] : [data.azurerm_monitor_action_group.email.id, data.azurerm_monitor_action_group.slack.id]
```

Comment:
why not `local.action_group`?

Reply:
Yes, this can be changed with local variable. Replaced with the variable `local.action_group`.